jenkins/internal-ci/generic/nightly/nightly-deploy.groovy

```diff
@@ -152,7 +152,8 @@ pipeline {
                         env.Sanity_Failed = qaSanity.buildVariables.Sanity_Failed
                         env.Sanity_status = qaSanity.buildVariables.Sanity_Failed.toString() == 'true' ? 'Failed' : qaSanity.buildVariables.Sanity_Failed.toString() == 'false' ? 'Passed' : 'Skipped'
                         env.Regression_Failed = qaSanity.buildVariables.Regression_Failed
-                        env.Regression_status = qaSanity.buildVariables.Regression_Failed.toString() == 'true' ? 'Failed' : qaSanity.buildVariables.Regression_Failed.toString() == 'false' ? 'Passed' : 'Skipped'
+                        env.Regression_overall_failed = = qaSanity.buildVariables.Regression_overall_failed
```

Comment:
This assignment is not require. As on L156 assigning it.

Reply:
Done

Reply:
Looks good to me